spacialgaze-plugins/SG.js

```diff
@@ -116,6 +116,24 @@ SG.reloadCSS = function () {
 	http.get(options);
 };
 
+//Daily Rewards System for SpacialGaze by Lord Haji
+SG.giveDailyReward = function (userid, user) {
+	if (!user || !userid) return false;
+	userid = toId(userid);
+	if (!user.autoconfirmed) return false;
+	if (!Db('DailyBonus').has(userid)) Db('DailyBonus').set(userid, 1);
+	if (!Db('BonusTime').has(userid)) Db('BonusTime').set(userid, 0); //if the user is connected for the first time...
+	let lastTime = Db('BonusTime').get(userid);
```

Comment:
We should try to not use 2 json files for this, its easy enough to use a object or array that contains both the time and the streak. Example:

```javascript
Db('DailyBonus').set(userid, [1, Date.now()]);

Db('DailyBonus').get(userid); // => [1, 1485099942220] (example output)
```